comfy_cli/cmdline.py

```diff
@@ -67,6 +69,46 @@ def init():
     print(f"scan_dir took {end_time - start_time:.2f} seconds to run")
 
 
+@app.command(help="Backup current snapshot")
+@tracking.track_command()
+def backup(
+    ctx: typer.Context,
+    output: Annotated[
+        str,
+        "--output",
+        typer.Option(show_default=False, help="Specify the output file path. (.yaml)"),
+    ],
+):
+
+    if not output.endswith(".yaml"):
+        print(f"[bold red]The output path must end with '.yaml'.[/bold red]")
+        raise typer.Exit(code=1)
+
+    output_path = os.path.abspath(output)
+
+    config_manager = workspace_manager.config_manager
```

Comment:
config_manager is also singleton, you can safely initiating it with `ConfigManager()`